dune-project

```diff
@@ -1,6 +1,9 @@
-(lang dune 2.7)
+(lang dune 3.7)
 (name semgrep)
 (using menhir 2.1)
+; disable mangling of of workspace root, to preserve debugging information
```

Comment:
can you add a link to a doc explaning what it is?

Reply:
Done